internal/driver/driver.go

```diff
@@ -342,3 +343,9 @@ func valueExtractor(ix int) sampleValueFunc {
 		return v[ix]
 	}
 }
+
+func addPseudoLocs(p *profile.Profile, cfg config, ui plugin.UI) {
+	tr, tl := p.AddPseudoLocs(cfg.TagRoot, cfg.TagLeaf)
```

Comment:
I'd prefer this method to be an internal method. We try to give higher guarantees about what's in the public profile package and here I am not sure we want to commit to maintain this API surface.

Reply:
Can you clarify what you mean by internal?  Should I move this PR to a Google-internal CL?